RecoBTag/TensorFlow/python/pfMassIndependentDeepDoubleBvLJetTags_cfi.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+pfMassIndependentDeepDoubleBvLJetTags = cms.EDProducer('DeepDoubleXTFJetTagsProducer',
```

Comment:
similarly, appropriate cloning instead of full cfi copy-paste is a better solution